service/src/main.rs

```diff
@@ -198,68 +214,40 @@ fn main() {
 			smatches.is_present("skip-ra"),
 		);
 	} else if matches.is_present("shielding-key") {
```

Comment:
I left all the separate calls in place for now (`shielding-key`, `signing-key`, `init-shard`, `request-state`). But they are not used anymore by our test setup. Maybe it would make sense in the future to remove them completely?

Reply:
I agree that request-state is not necessary any more. However, if you'd like to initialize multiple shards for example, then the separate `init-shard` is still necessary I believe?

Reply:
Yes, so if it still has a use-case, we should keep it.

Reply:
I don't think we need to dump the shielding- or signing key. This is basically legacy from the first few 100 lines of code :D.

But I'd like to see the enclave account clearly in the startup log.